src/lib/receiver.js

```diff
@@ -24,6 +24,7 @@ const BigNumber = require('bignumber.js')
  * @param  {ilp-core.Client} [opts.client] [ilp-core](https://github.com/interledger/js-ilp-core) Client, which can optionally be supplied instead of the previous options
  * @param  {Buffer} [opts.hmacKey=crypto.randomBytes(32)] 32-byte secret used for generating request conditions
  * @param  {Number} [opts.defaultRequestTimeout=30] Default time in seconds that requests will be valid for
+ * @param  {Boolean} [opts.disallowOverPayment=false] Require that incoming transfer amounts exactly match the requested amount
```

Comment:
I would reverse this: `allowOverPayment=false`.

I get that overpayment is theoretically good for the recipient, but in practice isn't it just going to mean that some error happened and if you don't decline the payment you're going to have to manually go in an fix it later?

At least in ITP it seems like overpayment would be an indicator of a serious problem and should be rejected. And if somebody truly needs it they can set `allowOverPayment`.

Also, I think it should be `allowOverpayment`, see [overpayment](http://dictionary.cambridge.org/dictionary/english/overpayment).